### DISCUSSION TITLE
HoYo launcher - starrail

### DISCUSSION THREAD

**Discussion by defmetal** [moraroy/NonSteamLaunchers-On-Steam-Deck#371] [General]
The hoyo launcher installs all hoyo games just fine like Star Rail and ZZZ and Genshin, but Star Rail won't launch. Any way to get this game working? It was kind of the one I was hoping would work but does not launch.  ZZZ launches just fine though

**Comment by moraroy** [comment]
@defmetal that game is blocked on Linux haha may have to Google that one

**Comment by defmetal** [comment]
But their other games on the same engine work. Like genshin came out before it and it works fine, and ZZZ came out last month and it works. So why only star rail?

**Reply by moraroy** [reply]
I'm not sure , but as far as I know they banned it on linux
